llm_mlx.py

```diff
@@ -188,3 +270,46 @@ def execute(self, prompt, stream, response, conversation):
             "peak_memory": chunk.peak_memory,
             "finish_reason": chunk.finish_reason,
         }
+
+
+def select_models(model_choices):
+    selected = [False] * len(model_choices)
+    idx = 0
+    window_size = os.get_terminal_size().lines - 5
+    
+    while True:
+        print("\033[H\033[J", end="")
```

Comment:
Normally I'd suggest we use a cross-platform library like Rich here, but since this plugin only works on macOS there's no need for us to add another dependency.